### DISCUSSION TITLE
pages/UA-PNP/Docs-as-Code/about/

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#8] [Q&A]
# pages/UA-PNP/Docs-as-Code/about/

A next-gen documentation site for HCL <product name>.

https://github01.hclpnp.com/pages/UA-PNP/Docs-as-Code/about/

**Comment by syeda-git** [comment]
test comment

**Reply by syeda-git** [reply]
response comment

**Comment by vishwanathan-m1** [comment]
Heading in this page needs to be formatted.
**Actual**: About {{prodname}} documentation pages. 
_Expected_: About product name documentation pages.
> Note: This applies to all the pages.

**Reply by vishwanathan-m1** [reply]
Fixes done. Please check and confirm. 

Thanks,
**Vishwa**

**Reply by vishwanathan-m1** [reply]
* Looks good and It can be closed. 
* Great Work !
